.pre-commit-config.yaml

```diff
@@ -13,3 +13,8 @@ repos:
     hooks:
     -   id: black
         exclude: ^(dandi/_version\.py|versioneer\.py)$
+-   repo: https://github.com/PyCQA/isort
+    rev: 5.7.0
+    hooks:
+    -   id: isort
+        exclude: ^(dandi/_version\.py|versioneer\.py)$
```

Comment:
also exclude `dandi/due.py` please

Reply:
Do you want that to be excluded from black as well?

Reply:
ideally - yes, I thought it was so.  It is a "standard" stab file from duecredit, we might take a new version in some time in the future.  I guess I have missed adding it to exclusions when I have added it.